config/environments/staging.rb

```diff
@@ -0,0 +1,107 @@
+Rails.application.configure do
```

Comment:
I added this because without it Rails uses wrong defaults with security vulnerabilities on Heroku.